runtime/common/src/assigned_slots/mod.rs

```diff
@@ -77,7 +81,11 @@ type LeasePeriodOf<T> =
 pub mod pallet {
 	use super::*;
 
+	/// The current storage version.
+	const STORAGE_VERSION: StorageVersion = StorageVersion::new(2);
```

Comment:
I believe `STORAGE_VERSION` should be set to `1` here, but would prefer somebody else to confirm or deny this.